src/doc/functional/_enrollment.rst

```diff
@@ -7,119 +7,90 @@ Enrollment Services
 Services
 """"""""
 
-.. py:function:: createEnrollment(enrollmentID, enrollmentData, transactionID)
+.. py:function:: createEnrollment(enrollmentID, enrollmentTypeId, enrollmentFlags, requestData, biometricData, biographicData, documentData, transactionID)
     :noindex:
 
     Insert a new enrollment.
 
     **Authorization**: :todo:`To be defined`
 
     :param str enrollmentID: The ID of the enrollment. If the enrollment already exists for the ID an error is returned.
-    :param dict enrollmentData: The enrollment attributes.
+    :param str enrollmentTypeId: The enrollment type ID of the enrollment.
+    :param dict enrollmentFlags: The enrollment custom flags.
+    :param dict requestData: The enrollment data related to the enrollment itself.
+    :param dict biometricData: The enrollment biometric data.
+    :param dict biographicData: The enrollment biographic data.
+    :param dict documentData: The enrollment biometric data.
     :param string transactionID: The client generated transactionID.
     :return: a status indicating success or error.
 
-.. py:function:: readEnrollment(enrollmentID, filter, transactionID)
+.. py:function:: readEnrollment(enrollmentID, attributes, transactionID)
     :noindex:
 
     Retrieve the attributes of an enrollment.
 
     **Authorization**: :todo:`To be defined`
 
     :param str enrollmentID: The ID of the enrollment.
-    :param set filter: The (optional) set of required attributes to retrieve.
+    :param set attributes: The (optional) set of required attributes to retrieve.
     :param string transactionID: The client generated transactionID.
     :return: a status indicating success or error and in case of success the enrollment data.
 
-.. py:function:: updateEnrollment(enrollmentID, enrollmentData, transactionID)
+.. py:function:: updateEnrollment(enrollmentID, enrollmentTypeId, enrollmentFlags, requestData, biometricData, biographicData, documentData, transactionID)
```

Comment:
Could you also add a "partialUpdateEnrollment" service (I think we agreed we needed both, like in PR interface)